src/client/components/notification/notification.stories.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import Notification from './notification.component';
+import { withKnobs, text } from '@storybook/addon-knobs';
+
+export default {
+  title: 'Notification',
+  component: Notification,
+  decorators: [withKnobs],
+};
+
+export const Error = () => {
+  const txt = text(
+    'Error text',
+    'This is error message for wrong email address.',
+  );
+  return <Notification text={txt} />;
```

Comment:
```suggestion
  const errorText = text(
    'Error text',
    'This is error message for wrong email address.',
  );
  return <Notification text={errorText} />;
```

I'm nitpicking a bit, but really don't like this types of "deliberate misspellings" because a given name is already used for something else. Just make longer names and let your editor autocomplete for you, it's easier to read :)

Reply:
If you're lazy this is also an option, but I like the pattern of giving knobs their own variables.

```suggestion
  return <Notification text={
     text(
      'Error text',
      'This is error message for wrong email address.',
    )
  } />;
```

Reply:
Isn't "txt" acronym for text, as "btn" is for button, but yes I can put longer name.